docs/root/api-docs/xds_protocol.rst

```diff
@@ -550,10 +566,10 @@ to. For example, if the client had previously been subscribed to resources A and
 unsubscribe from B, it must send a new request containing only resource A.
 
 Note that for :ref:`Listener <envoy_v3_api_msg_config.listener.v3.Listener>` and :ref:`Cluster <envoy_v3_api_msg_config.cluster.v3.Cluster>`
-resource types where the stream is in "wildcard" mode (see :ref:`How the client specifies what
+resource types where the client is using a "wildcard" subscription (see :ref:`How the client specifies what
 resources to return <xds_protocol_resource_hints>` for details), the set of resources being
-subscribed to is determined by the server instead of the client, so there is no mechanism
-for the client to unsubscribe from resources.
+subscribed to is determined by the server instead of the client, so the client cannot unsubscribe
+from those resources individually; it can only unsubscribe from the wildcard as a whole.
```

Comment:
could you elaborate how to effectively unsubscribe resources that are "included " in the "wildcard subscription" when the client unsubscribes "*"?

Reply:
I'd say that the client should be more or less able to track which resources are from wildcard subscription, so it should be able to drop them from its caches when unsubscribing from `*`.

Reply:
Yes, @krnowak is correct.  The client already knows which specific non-wildcard resources it is subscribed to, so when it unsubscribes from `*`, it can immediately drop all other resources from its cache.